app/[locale]/projects/page.tsx

```diff
@@ -83,32 +84,37 @@ const ProjectsPage = () => {
         filter,
       });
 
-      setProjects(
-        projects.filter((p: Project) =>
-          p.item.data.repository.name
-            .toLocaleLowerCase()
-            .trim()
-            .includes(searchByProjectNameValue.toLocaleLowerCase().trim())
-        )
+      const projecteFilterdByName = projects.filter((p: Project) =>
+        p.item?.name?.toLocaleLowerCase()
+          .trim()
+          .includes(searchByProjectNameValue.toLocaleLowerCase().trim())
```

Comment:
If you are not passing any param to `toLocaleLowerCase` will it be any different than the `toLowerCase` name?

Reply:
Hi @Tamir198,

Good point. In this case, since we are not passing any locale-specific parameters to toLocaleLowerCase(), it behaves the same as toLowerCase(). I’ll update the code to use toLowerCase() for simplicity.